setup.cfg

```diff
@@ -0,0 +1,10 @@
+
+[flake8]
+max-line-length = 100
```

Comment:
Yeah the formatter black actually uses something like 88 by default (and pep 8 says 80 I think), so  100 was a bit of a compromise so that the code didn't change too much. But I'm happy to change it in future, the important thing is that everyone uses the same number!